pkg/operator/ceph/object/objectstore.go

```diff
@@ -916,11 +916,14 @@ func adjustZoneDefaultPools(zone map[string]interface{}, spec cephv1.ObjectShare
 		"user_uid_pool":   ".meta.users.uid",
 		"otp_pool":        ".otp",
 		"notif_pool":      ".log.notif",
+		"topics_pool":     ".meta.topics",  // introduced in Ceph v19
```

Comment:
If we try to set these new properties in v18, does it fail? I assume that's why you're checking if the properties exist like this.

Reply:
Correct, if pool is not presented in source json then we just ignore it.